src/shogun/machine/BaggingMachine.cpp

```diff
@@ -177,7 +178,8 @@ bool CBaggingMachine::train_machine(CFeatures* data)
 	for (index_t i = 0; i < m_num_bags*m_bag_size; ++i)
 		rnd_indicies.matrix[i] = CMath::random(0, m_bag_size-1);
 
-	#pragma omp parallel for
+	auto pb = progress(range(m_num_bags));
```

Comment:
@geektoni @shubham808 does the progress bar work gracefully from within omp?

Reply:
Yes, it does :)